myuw/static/vue/components/teaching/course/materials/email-modals/request.vue

```diff
@@ -1,14 +1,11 @@
 <template>
   <b-modal
     :id="`emaillist_request_${sln}`"
-    v-model="show"
+    ref="request-modal"
     title="Create Mailing List"
+    @hidden="onHide()"
   >
-    <b-alert variant="danger" :show="disableActions">
-      This action is disabled while overriding as another user.
-    </b-alert>
-
-    <div v-if="emailList.request_sent">
+    <template v-if="emailList.request_sent">
       <b-alert variant="success" :show="disableActions">
```

Comment:
How could request_sent when disableActions is true?
Why this alert depend on disableActions?